src/passes/MinifyImportsAndExports.cpp

```diff
@@ -150,20 +151,34 @@ struct MinifyImportsAndExports : public Pass {
     };
     ModuleUtils::iterImportedGlobals(*module, processImport);
     ModuleUtils::iterImportedFunctions(*module, processImport);
-    // Minify the exported names.
-    for (auto& curr : module->exports) {
-      process(curr->name);
+
+    if (minifyExports) {
+      // Minify the exported names.
+      for (auto& curr : module->exports) {
+        process(curr->name);
+      }
     }
     module->updateMaps();
     // Emit the mapping.
     for (auto& pair : oldToNew) {
       std::cout << pair.second.str << " => " << pair.first.str << '\n';
     }
   }
+
+public:
+  explicit MinifyImportsAndExports(bool minifyExports)
```

Comment:
for consistency with code style elsewhere, please put this right after the new bool that was added at the top, and format it as
```
MinifyImportsAndExports(bool minifyExports) : minifyExports(minifyExports) {}
```